src/d5a50_len_5fd0.c

```diff
@@ -68,13 +70,40 @@ void draw_hud_element_clipped(s32 arg0) {
     draw_hud_element(arg0, 0);
 }
 
-INCLUDE_ASM(s32, "d5a50_len_5fd0", draw_hud_element_2);
+void draw_hud_element_2(s32 arg0) {
+    draw_hud_element(arg0, 1);
+}
 
 void draw_icon_2(s32 iconID) {
     draw_hud_element(iconID, 2);
 }
 
-INCLUDE_ASM(s32, "d5a50_len_5fd0", set_hud_element_script);
+void set_hud_element_script(s32 arg0, s32 *arg1) {
+    HudElement *hudElement;
```

Comment:
Pointer on the left (`HudElement*`) and you can initialize this on the same line you declare it